src/components/Person/Person.jsx

```diff
@@ -1 +1,39 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    sex,
+    age,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerSex = sex === 'm'
```

Comment:
```suggestion
  const partnerStatus = sex === 'm'
```
Be more specific on what value you store.